src/test/java/racingcar/domain/RoundTest.java

```diff
@@ -0,0 +1,42 @@
+package racingcar.domain;
+
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.junit.jupiter.api.Assertions.*;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+class RoundTest {
+
+    @DisplayName("라운드 값이 정수가 아니면 예외를 던진다")
+    @ParameterizedTest
+    @ValueSource(strings = {"조조는바보", "감자는멋져", "초코12", "sdfsd", " ", ""})
+    void testRoundCountIsNotInteger(String roundCount) {
+        assertThatThrownBy(() -> new Round(roundCount))
+                .isInstanceOf(IllegalArgumentException.class);
+    }
+
+    @DisplayName("라운드 값이 1보다 작으면 예외를 던진다")
+    @ParameterizedTest
+    @ValueSource(strings = {"0", "-1", "-2", "-3"})
+    void testInvalidRoundCountRange(String roundCount) {
+        assertThatThrownBy(() -> new Round(roundCount))
+                .isInstanceOf(IllegalArgumentException.class);
+    }
+
+    @DisplayName("라운드 값이 1이상 정수이면 예외를 던지지 않는다")
+    @ParameterizedTest
+    @ValueSource(strings = {"1", "2", "100", "101"})
+    void testValidRoundCount(String roundCount) {
+        assertDoesNotThrow(() -> new Round(roundCount));
+    }
+
+    @DisplayName("라운드가 양수이면 참을 반환한다")
```

Comment:
어떤 테스트인지 알기 어려운 설명인거 같아요!
```suggestion
    @DisplayName("라운드가 양수이면 isRemain()함수가 참을 반환한다")
```
라던지, `라운드가 양수이면 라운드는 남아있다` (좀  이상하긴 하네요..) 같이 어떤 테스트인지 설명이 보강되면 좋겠어요

Reply:
아래 내용으로 수정했습니다.
```
    @DisplayName("라운드 값이 남아 있으면 참을 반환한다")
    @ParameterizedTest
    @ValueSource(strings = {"1", "2", "3", "4", "100"})
    void testIsRemain(String roundCount) {
```

Reply:
어떤 함수가 영향을 받는지 분명히 해야한다는 의미였습니당!
보통 테스트 코드를 모두 보면서 테스트를 돌리지 않으니, 다른 개발자가 이 displayName에서 뜨는 설명만 보고 해당 테스트가 어떤 테스트인지 파악되어야 합니다.

아래 이미지 처럼 설명이 나오면 테스트 코드로 들어오지 않고선 내용을 알 수 없습니다
<img width="470" alt="image" src="https://github.com/woowacourse/java-racingcar/assets/51393021/bb827304-5d78-486a-a530-4347cfc3dd1a">